vuepress/docs/v6.3.2/docs/getting-started/README.md

```diff
@@ -158,19 +158,19 @@ sudo kubectl create namespace entando
 #### Add Custom Resources
 
 ::: tip Why Custom Resources?
-Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access your pods, and `Ingresses`, for managing external access to your cluster.
+Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access pods, and `Ingresses`, for managing external access to clusters.
 
-[Custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) let you store and retrieve structured data. Combining a custom resource with a custom controller allows you to define a desired state to automate the running of your applications or services in a Kubernetes cluster.
+[Custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) let you retrieve and store structured data. Combining a custom resource with a custom controller allows you to define a desired state to automate the running of your applications or services in a Kubernetes cluster.
```

Comment:
Typically the order is store and retrieve, because you can't retrieve something that isn't stored. The shorthand in requirements analysis is CRUD - Create, Retrieve, Update, Delete - and the features are usually implemented in that order.

Reply:
ha! thank you for the explanation :) i was thinking that you need to retrieve something to store it... i'll revert.